src/state/auth/actions.js

```diff
@@ -58,3 +59,11 @@ export function logUserOut() {
     });
   };
 }
+
+export function setAuthStarted(started = true) {
```

Comment:
I would go into `/src/utils/auth0.js` and change the two exported function's names to `authorize` and `parseHash` since they are really wrappers on top of those functions.

Then I would probably change this to:
```js
import { authorize, parseHash } from 'console/utils/auth0.js';

export function startAuthenticationFlow(returnUrl) {
  return dispatch => {
    authorize(returnUrl);
    return dispatch({
      type: USER_AUTH_START,
    });
  };
}

export function finishAuthenticationFlow() {
  return dispatch => {
    return dispatch({
      type: USER_AUTH_FINISH,
    });
  }
}
```

Above in this same file `loginFailed` should change to:
```js
  export function authenticationFailed(error) {
    return dispatch => {
      dispatch(finishAuthenticationFlow());
      return dispatch({
        type: USER_AUTH_FAILURE,
        error,
      });
    }
  }
```

And `QueryAuth0` will need an update in `componentDidMount`:
```js
try {
  authResult = await parseHash();
} catch (err) {
  loginFailed(err);
}

if (authResult) {
  finishAuthenticationFlow();
} else {
 ...
```